### PR TITLE
Setup directories for Metrics validation and verification KEP

### PR DESCRIPTION
**Pull request by serathius**
This PR add directories and setups directory OWNERS for implementation of KEP

Issue: https://github.com/kubernetes/enhancements/issues/1209

/kind feature
/sig instrumentation

```release-note
NONE
```

```docs
- [KEP]: https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/20190605-metrics-validation-and-verification.md
```
/cc @logicalhan 
FYI @danielmk

### CODE REVIEW

**Comment by serathius** [comment]
Cannot set `sig-instrumentation-pr-reviews` as reviewers, there is no record in https://github.com/kubernetes/kubernetes/blob/master/OWNERS_ALIASES for it

**Review by logicalhan — APPROVED** [review]
/lgtm

File: test/instrumentation/testdata/OWNERS
Comment (liggitt) [inline test/instrumentation/testdata/OWNERS]: can you clarify why this is exempted from the normal hierarchy? the reason we've typically done that is for cross-cutting things like API approval, but have tied to an alias rather than specific approvers

**Comment by cblecker** [comment]
/lgtm cancel
/hold

Has sig-testing been engaged on this at all? We're listed as participating in the KEP, but I'm not sure who exactly is involved. Also, not comfortable with a no_parent_owners here.. as @liggitt pointed out, this should only be used in specific circumstances (like API approvals)

File: test/instrumentation/testdata/OWNERS
Comment (logicalhan) [reply test/instrumentation/testdata/OWNERS]: An alias for this is better. We should probably use `sig-instrumentation-leads`. This directory should be exempted from the normal hierarchy because it functions as a cross-cutting API approval type mechanism for stable metrics. This directory (and the file which will live in it) will be functionally identical to `test/conformance/testdata`.

**Comment by BenTheElder** [comment]
I think I'm going to let @liggitt (as a test/ and root-level approver...) approve / no-approve here instead as I'm not so sure on this, but +1 to @cblecker's comment ...
/assign @liggitt
/unassign

File: test/instrumentation/testdata/OWNERS
Comment (serathius) [reply test/instrumentation/testdata/OWNERS]: @liggitt Like Han said, goal is to have directory with list of stable metrics that will be reviewed by sig-instrumentation

Comment (cblecker) [reply test/instrumentation/testdata/OWNERS]: I'm an explicit -1 on `no_parent_owners ` at this time. Both API approvals and conformance testing are under the purview of sig-architecture, who is specifically delegated authority to gate on project-wide design decisions.

I think the regular hierarchy should apply here, with instrumentation being the owners of this directory.

Comment (serathius) [reply test/instrumentation/testdata/OWNERS]: @brancz WDYT?

Comment (serathius) [reply test/instrumentation/testdata/OWNERS]: Removed it for now to speed up review process.

**Comment by liggitt** [comment]
lgtm as a location for instrumentation test tools/data. will let sig-testing folks ack in its current state.

**Comment by serathius** [comment]
Can someone from sig-testing approve?

**Comment by cblecker** [comment]
/lgtm
/approve

I'm good with this directory now that the `no_parent_owners` has been removed. I still thing that sig-intrumentation and sig-testing need to connect on the KEP itself, but it shouldn't block this directory.

**Pull request merged**
